fedimint-recurringd/src/lib.rs

```diff
@@ -405,3 +498,19 @@ async fn await_invoice_confirmed(
 pub enum PaymentCodeInvoice {
     Bolt11(Bolt11Invoice),
 }
+
+/// Helper struct indicating if an invoice was paid. In the future it may also
+/// contain the preimage to be fully LUD-21 compliant.
+pub struct InvoiceStatus {
+    pub invoice: Bolt11Invoice,
+    pub paid: bool,
```

Comment:
nit: bools are generally meh, might be good to just make this an enum for if you do want to add the preimage in the future